Source/DafnyLanguageServer/Language/Symbols/SymbolTableFactory.cs

```diff
@@ -351,8 +400,8 @@ private class SymbolDeclarationLocationVisitor : ISymbolVisitor<Unit> {
         }
       }
 
-      private void RegisterLocation(ISymbol symbol, IToken token, Range name, Range declaration) {
-        if (token.filename != null) {
+      private void RegisterLocation(ISymbol symbol, IToken? token, Range name, Range declaration) {
```

Comment:
I believe this is some leftover of the refactorings you did. Or is it necessary that `token` is nullable?